src/DirectoryReader.cpp

```diff
@@ -63,21 +63,21 @@ DirectoryReader::Entry DirectoryReader::Next() {
    Entry entry = std::make_pair(FileType::Unknown, "");
    bool found = false;
    while (!found && (entry.first != FileType::End)) {
-      readdir64_r(mDirectory, &mEntry, &mResult); // readdir_r is reentrant
+      mResult = readdir64(mDirectory);
```

Comment:
NOTE: readdir64_r has been deprecated. readdir64 is thread safe.
See https://man7.org/linux/man-pages/man3/readdir_r.3.html